src/main/java/com/akademia/projectplanner/enums/Message.java

```diff
@@ -0,0 +1,18 @@
+package com.akademia.projectplanner.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/** An enumeration of messages that are displayed when throwing an exception. */
+@AllArgsConstructor
+@Getter
+public enum Message {
```

Comment:
Trzeba zmienić nazwę ponieważ klasa o nazwie Message istnieje już w Javie, więc może być to mylące - jak spojrzysz w IDE to podświetla nawet nazwę pola "private String message" i jak najedziesz myszą i klikniesz w podpowiesz SonarLint'a to będzie cały opis przypadku. Jeśli nie masz tego to trzeba zainstalować wtyczkę SonarLint (daj znać). Moje propozycje to coś w stylu ExcepitionMessage, ExceptionText